Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Search.hs

```diff
@@ -141,7 +143,8 @@ handler merchant sReq = do
   let fromLocationLatLong = sReq.pickupLocation
       toLocationLatLong = sReq.dropLocation
       merchantId = merchant.id
-  result <- getDistanceAndDuration merchantId fromLocationLatLong toLocationLatLong sReq.routeDistance sReq.routeDuration
+  merchantOperatingCity <- SMOC.findByMerchantId merchantId >>= fromMaybeM (MerchantOperatingCityNotFound merchantId.getId)
```

Comment:
We don't need to fetch the merchantOperatingCity until we use the values inside it.

Reply:
> We don't need to fetch the merchantOperatingCity until we use the values inside it.
@hkmangla  we are using it in this function.